symplyphysics/laws/waves/blackbody_radiation/wien/spectral_energy_density.py

```diff
@@ -0,0 +1,60 @@
+from sympy import Eq, exp, pi
```

Comment:
It seems that you prefer to separate equations by marketing names, eg `planck` or `wien`. But how is user supposed to find proper equation for his needs? Can we somehow describe, how they are different from each other? And we can use this difference as folder name.

Reply:
I'm not sure how to adequately name the folders so that they convey the "law works correctly for high/low/any frequencies" meaning

Reply:
We can also get rid of folders entirely and just mention the distinction in file names.

Reply:
Good idea, if we expect just one related law.